Unable to build Android editor with `dev_build=yes scu_build=yes`

### Tested versions

Reproducible: Latest master (5da6deaac), 4.4 dev 6
Not reproducible: 4.4 dev 5

### System information

Godot v4.4.beta (5da6deaac) - Windows 10 (build 19045) - Multi-window, 2 monitors - OpenGL 3 (Compatibility) - AMD Radeon RX 580 2048SP (Advanced Micro Devices, Inc.; 31.0.21921.1000) - AMD Ryzen 5 3600 6-Core Processor (12 threads)

### Issue description

Attempting to compile Godot with `dev_build=yes scu_build_yes target=editor platform=android` results in this errors after some time:

```
[ 44%] Compiling thirdparty\embree\common\lexers\tokenstream.cpp ...
[ 44%] Compiling thirdparty\embree\common\tasking\taskschedulerinternal.cpp ...
ERROR: In file included from modules\openxr\extensions\scu\scu_modules_openxr_extensions.gen.cpp:9:
.\modules/openxr/extensions/openxr_fb_update_swapchain_extension.cpp:58:22: error: use of undeclared identifier 'XR_FB_SWAPCHAIN_UPDATE_STATE_VULKAN_EXTENSION_NAME'
                request_extensions[XR_FB_SWAPCHAIN_UPDATE_STATE_VULKAN_EXTENSION_NAME] = &fb_swapchain_update_state_vulkan_ext;
                                   ^
.\modules/openxr/extensions/openxr_fb_update_swapchain_extension.cpp:62:22: error: use of undeclared identifier 'XR_FB_SWAPCHAIN_UPDATE_STATE_OPENGL_ES_EXTENSION_NAME'
                request_extensions[XR_FB_SWAPCHAIN_UPDATE_STATE_OPENGL_ES_EXTENSION_NAME] = &fb_swapchain_update_state_opengles_ext;
                                   ^
2 errors generated.

[ 45%] Compiling thirdparty\embree\kernels\common\device.cpp ...
scons: *** [modules\openxr\extensions\scu\scu_modules_openxr_extensions.gen.android.editor.dev.arm64.o] Error 1
scons: building terminated because of errors.
INFO: Time elapsed: 00:00:36.33
```

<br>

This seem a regression from #98261 that added the problematic file to SCU but that was hidden because in 4.4 dev 4 we also had pr #98888 that broke SCU on Windows. Only after #99629 that fixed SCU on Windows the bug appeared (and almost made me blame the wrong person because the bisect pointed to this pr hahaha).

### Steps to reproduce

Compile Godot with `dev_build=yes scu_build_yes target=editor platform=android`

### Minimal reproduction project (MRP)

N/A

--- a/scu_builders.py
+++ b/scu_builders.py
@@ -326,7 +326,7 @@ def generate_scu_files(max_includes_per_scu):
     process_folder(["modules/openxr"], ["register_types"])
     process_folder(["modules/openxr/action_map"])
     process_folder(["modules/openxr/editor"])
-    process_folder(["modules/openxr/extensions"])
+    # process_folder(["modules/openxr/extensions"])  # Sensitive include order for platform code.
     process_folder(["modules/openxr/scene"])
     process_folder(["modules/godot_physics_2d"])
     process_folder(["modules/godot_physics_3d"])

